### PR TITLE
Add season3 videos

### PR DESCRIPTION
**Pull request by lwandzura**
Closes https://github.com/influxdata/docs-v2/issues/3821

Added all Season 3 videos to the Resources page.

Added a description for each video and added related links to relevant docs.

Added the videos to corresponding topic docs.

### CODE REVIEW

File: content/telegraf/v1.21/introduction/getting-started.md [hunk 1]
```diff
@@ -149,6 +149,11 @@ Notice that the timestamps occur at rounded ten second intervals (that is, `:00`
...
 That's it! You ready to use Telegraf to collect metrics and write them to your output of choice.
 
-For an introduction to Telegraf and an overview of how it works, watch the following video:
+For an introduction to Telegraf and an overview of how it works, watch the following videos:
+
+{{< youtube vGJeo3FaMds >}}
+
+
+{{< youtube k5m7SHgXRYQ >}}
+
```
Comment (sanderson) [inline content/telegraf/v1.21/introduction/getting-started.md:158]: Two large videos right next to each other feels weird to me. Maybe put them in tabs?

```suggestion
For an introduction to Telegraf and an overview of how it works, watch the following videos:

{{< tabs-wrapper >}}
{{% tabs "small" %}}
[Intro to Telegraf](#)
[Telegraf Plugin Overview](#)
{{% /tabs %}}
{{% tab-content %}}
{{< youtube vGJeo3FaMds >}}
{{% /tab-content %}}
{{% tab-content %}}
{{< youtube k5m7SHgXRYQ >}}
{{% /tab-content %}}
{{< /tabs-wrapper >}}
```

**Review by sanderson — CHANGES_REQUESTED** [review]

File: content/flux/v0.x/stdlib/universe/join.md [hunk 2]
```diff
@@ -218,3 +218,7 @@ union(tables: [t1, t2])
 | 2021-01-03T00:00:00Z | foo |      3 |
 | 2021-01-04T00:00:00Z | foo |      4 |
 {{% /expand %}}
+
+To see how join works in InfluxDB, watch the following video:
+
+{{< youtube g-HY1Q11iGg >}}
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/flux/v0.x/stdlib/universe/join.md:224]: @sanderson good point!  Will move the video

File: content/flux/v0.x/stdlib/universe/pivot.md [hunk 3]
```diff
@@ -181,3 +194,7 @@ sampledata.int()
 {{< /flex >}}
 {{% /expand %}}
 {{< /expand-wrapper >}}
+
+To see an overview of the pivot function, watch the following video:
+
+{{< youtube LuPcShbE0J4 >}}
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/flux/v0.x/stdlib/universe/pivot.md:200]: removed video from stdlib

File: content/flux/v0.x/stdlib/universe/pivot.md [hunk 4]
```diff
@@ -62,9 +62,22 @@ Default is piped-forward data ([`<-`](/flux/v0.x/spec/expressions/#pipe-expressi
...
+    - [Align fields within each measurement that have the same timestamp](#align-fields-within-each-measurement-that-have-the-same-timestamp)
+      - [Input data](#input-data)
+      - [Output data](#output-data)
+    - [Align fields and measurements that have the same timestamp](#align-fields-and-measurements-that-have-the-same-timestamp)
+        - [Note the effects of:](#note-the-effects-of)
+      - [Input data](#input-data-1)
+      - [Output data](#output-data-1)
+    - [Align values for each tag](#align-values-for-each-tag)
+      - [Input data](#input-data-2)
+      - [Output data](#output-data-2)
```
[the remark this answers is not in the document]
Comment (lwandzura) [reply content/flux/v0.x/stdlib/universe/pivot.md:80]: I did not make any changes to this page except for adding the video, which I have removed.  Will commit the changes suggested.

**Review by sanderson — APPROVED** [review]

**Pull request merged**